### ISSUE TITLE
Do you accept pull requests?

### ISSUE THREAD

**Issue by cfilipescu** [rust-embedded/rust-sysfs-pwm#17]
Looking to add capture support to the library and I was wondering if you accept pull requests.

**Comment by posborne** [comment]
> Looking to add capture support to the library and I was wondering if you accept pull requests.

Yep, PRs welcome and will be reviewed by members of the embedded-linux subteam.
